src/components/Result.tsx

```diff
@@ -38,6 +38,11 @@ const Result = () => {
               <th className={`${styles.th} ${styles.centered}`}>
                 {t('result.attempts')}
               </th>
+              {results.some((result) => result.question.referenceLink) ? (
```

Comment:
I'd prefer the [logical and operator syntax](https://react.dev/learn/conditional-rendering#logical-and-operator-) for the conditional rendering. Safes a few lines and we don't have the empty string in the code.